From 7660b3a1264380cdaacb028f82af53e5af7f3b23 Mon Sep 17 00:00:00 2001
From: Jean Trivelly <jean.trivelly@intel.com>
Date: Fri, 12 Apr 2013 16:12:46 +0200
Subject: [PORT FROM MAIN] wifi: Copy Header files to correct place for external rel.

BZ: 99426

In order to be able to compile wpa_supplicant_lib,
it is required to copy headers since supplicant_8 sources
are not available for external release. (because in /PRIVATE/ path)

Change-Id: Ic3e97478b556b3e412753365e7afa3a84aa949ce
Orig-Change-Id: I8b2526e6e8ce53538c9d620b515417bced2d7f7c
Signed-off-by: Jean Trivelly <jean.trivelly@intel.com>
---
 wpa_supplicant/Android.mk |   63 +++++++++++++++++++++++++++++++++++++++++++++
 1 files changed, 63 insertions(+), 0 deletions(-)

diff --git a/wpa_supplicant/Android.mk b/wpa_supplicant/Android.mk
index a0c97f0..06beeaf 100644
--- a/wpa_supplicant/Android.mk
+++ b/wpa_supplicant/Android.mk
@@ -1551,6 +1551,69 @@ LOCAL_SRC_FILES := $(OBJS)
 LOCAL_C_INCLUDES := $(INCLUDES)
 include $(BUILD_EXECUTABLE)
 
+include $(CLEAR_VARS)
+LOCAL_COPY_HEADERS_TO := wpa_supplicant_8/wpa_supplicant
+LOCAL_COPY_HEADERS :=\
+    config.h \
+    config_ssid.h \
+    wpa_supplicant_i.h \
+    android.config
+include $(BUILD_COPY_HEADERS)
+
+include $(CLEAR_VARS)
+LOCAL_COPY_HEADERS_TO := wpa_supplicant_8/src/common
+LOCAL_COPY_HEADERS :=\
+    src/common/defs.h \
+    src/common/ieee802_11_common.h \
+    src/common/ieee802_11_defs.h
+include $(BUILD_COPY_HEADERS)
+
+include $(CLEAR_VARS)
+LOCAL_COPY_HEADERS_TO := wpa_supplicant_8/src/drivers
+LOCAL_COPY_HEADERS :=\
+    src/drivers/android_drv.h \
+    src/drivers/driver.h \
+    src/drivers/linux_ioctl.h \
+    src/drivers/netlink.h \
+    src/drivers/nl80211_copy.h \
+    src/drivers/rfkill.h
+include $(BUILD_COPY_HEADERS)
+
+include $(CLEAR_VARS)
+LOCAL_COPY_HEADERS_TO := wpa_supplicant_8/src/eap_peer
+LOCAL_COPY_HEADERS :=\
+    src/eap_peer/eap_config.h
+include $(BUILD_COPY_HEADERS)
+
+include $(CLEAR_VARS)
+LOCAL_COPY_HEADERS_TO := wpa_supplicant_8/src/utils
+LOCAL_COPY_HEADERS :=\
+    src/utils/build_config.h \
+    src/utils/common.h \
+    src/utils/eloop.h \
+    src/utils/includes.h \
+    src/utils/list.h \
+    src/utils/os.h \
+    src/utils/radiotap.h \
+    src/utils/radiotap_iter.h \
+    src/utils/wpabuf.h \
+    src/utils/wpa_debug.h
+include $(BUILD_COPY_HEADERS)
+
+include $(CLEAR_VARS)
+LOCAL_COPY_HEADERS_TO := wpa_supplicant_8/src/wps
+LOCAL_COPY_HEADERS :=\
+    src/wps/wps_defs.h \
+    src/wps/wps.h
+include $(BUILD_COPY_HEADERS)
+
+include $(CLEAR_VARS)
+LOCAL_COPY_HEADERS_TO := wpa_supplicant_8/wpa_supplicant/src/wps
+LOCAL_COPY_HEADERS :=\
+    src/wps/wps_defs.h \
+    src/wps/wps.h
+include $(BUILD_COPY_HEADERS)
+
 ########################
 #
 #include $(CLEAR_VARS)
-- 
1.7.4.1

